qln/msghandler.go

```diff
@@ -53,6 +47,28 @@ func (nd *LitNode) PeerHandler(msg lnutil.LitMsg, q *Qchan, peer *RemotePeer) er
 
 }
 
+// BroadcastMessage sends a chat message to all connected websocket clients
+func (nd *LitNode) BroadcastChatMessage(msg lnutil.LitMsg) error {
+	chat, ok := msg.(lnutil.ChatMsg)
+	if !ok {
+		return fmt.Errorf("can't cast to chat message")
+	}
+
+	// get the current set of channels and save them, they may be deleted by litrpc
+	// before the iteration is over
+	var channels []chan lnutil.ChatMsg
```

Comment:
Because of this: https://github.com/mit-dci/lit/pull/56/files#diff-4cec2aab01ce8eee14372ee15be1af85R100
I'm thinking about a possible race condition where if I only use one loop the function will never end because if there are multiple users they may delete their old chan and make a new one so quickly that the loop never ends.
I tried to reference this in the comment.